tests/publiccloud/slem_basic.pm

```diff
@@ -56,20 +62,19 @@ sub run {
 
     select_serial_terminal();
 
-    if ($qam) {
+    if ($qam or $args->{my_instance}) {
```

Comment:
This `OR` is needed because when `qam=0` in Latest, instance was already created by `prepare_instance`.

Reply:
Block updated in PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19186